cylc/flow/scheduler.py

```diff
@@ -905,7 +905,8 @@ def get_command_method(self, command_name: str) -> Callable:
     def queue_command(self, command: str, kwargs: dict) -> None:
         self.command_queue.put((
             command,
-            tuple(kwargs.values()), {}
+            [],
+            kwargs,
```

Comment:
Bug fix: this was turning command kwargs into args, which could result in arg order errors.

Reply:
(And I've ended up having to fix it #5658 too; conflicts coming damn it).